djangosaml2/views.py

```diff
@@ -152,7 +153,7 @@ def get_next_path(self, request: HttpRequest) -> str:
         return next_path
 
     def unknown_idp(self, request, idp):
-        msg = f"Error: IdP EntityID {idp} was not found in metadata"
+        msg = f"Error: IdP EntityID {escape(idp)} was not found in metadata"
         logger.error(msg)
         return HttpResponse(msg.format("Please contact technical support."), status=403)
```

Comment:
Please, just remove `.format("Please contact technical support.")` because it's not actually formatted

Reply:
Ah, you're right. Good spot!

Looks like @peppelinux handled this in a follow-up PR: https://github.com/IdentityPython/djangosaml2/pull/370